tests/bootstrap.php

```diff
@@ -21,6 +25,85 @@
 if (file_exists(__DIR__ . '/../.env')) {
     $dotenv->load();
 }
-$dotenv->required(['API_URL', 'KBC_MANAGE_TOKEN', 'KBC_STORAGE_TOKEN', 'KBC_URL']);
+$dotenv->required(['API_URL', 'KBC_MANAGE_TOKEN', 'KBC_MANAGE_NOTIFY_TOKEN', 'KBC_STORAGE_TOKEN', 'KBC_URL']);
```

Comment:
What is KBC_MANAGE_NOTIFY_TOKEN for?

Reply:
It's not new, I just added it here to make the debugging easier in the future :)

https://github.com/keboola/sandboxes-api-php-client/blob/main/tests/ClientFunctionalTest.php#L229

Reply:
ah, thanks!